src/requestManager.cpp

```diff
@@ -115,9 +113,8 @@ void CRequestManager::cleanup(OdMap::iterator &itemIt)
     for (CUnknownObj::ObjectSourceList::iterator i = item.availableFrom.begin(); i != item.availableFrom.end(); ++i)
     {
         CNode *node = i->node;
-        if (node)
+        if (node != nullptr)
         {
-            i->clear();
             // LOG(REQ, "ReqMgr: %s cleanup - removed ref to %d count %d.\n", item.obj.ToString(), node->GetId(),
```

Comment:
you're right, we're not deleting the CNode object, just the pointer so this should be fine to take out.